### PR TITLE
Publishing wagmi package similar to zora-drop-contracts

### PR DESCRIPTION
**Pull request by oveddan**
Additionally remove the `broadcast` folder

### CODE REVIEW

File: package/chainConfigs.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+export const chainConfigs = {
+  [1]: {
```
Comment (iainnash) [inline package/chainConfigs.ts:2]: is this auto-generated?

if not, would make sense to use constant names for chain ids?

**Review by iainnash — APPROVED** [review]
Looks good except for comment

Thanks for deleting the old broadcast files.

I'm okay with leaving run-latest since it makes verifying easier with exact-match.

File: package/chainConfigs.ts (hunk 1 above)
Comment (oveddan) [reply package/chainConfigs.ts:2]: It's generated automatically by this script:
https://github.com/ourzora/zora-1155-contracts/pull/93/files#diff-d58480676bf837f4a98dacf641cfd02442c21ad0555ba3a53a8cda64cf8f5350

Constant names would be great - but it's just right now generating this from whats in chainConfigs folder

**Comment by iainnash** [comment]
Should be good to merge.

**Pull request merged**